src/tree_schema_internal.h

```diff
@@ -22,6 +22,7 @@
 #include "tree_schema.h"
 #include "xml.h"
 
+#define YIN_NS_URI "urn:ietf:params:xml:ns:yang:yin:1"
```

Comment:
Please add an indentation blank line between not connected macros

Reply:
OK